### PR TITLE
Make the VerneMQ status page prettier

### PR DESCRIPTION
**Pull request by codeadict**
* Add nicer styles.
* Compact the information a little bit to see all in the page.
* Make it more responsive.

screen shot 2018-12-28 at 1 59 52 pm

### CODE REVIEW

**Comment by larshesel** [comment]
I like this! What do you think @dergraf

**Comment by dergraf** [comment]
Good for me! I am trusting the design skills of @codeadict

**Comment by codeadict** [comment]
Don't beat that much on that 🤣

File: changelog.md [hunk 1]
```diff
@@ -66,6 +69,7 @@
 - Fix crash in bridge when calling `vmq-admin session show` by fixing the
   `vmq_ql` row initializer to handle plugin sessions (the bridge starts a local
   plugin session).
+>>>>>>> 93838e09778c8467768b4e5f33520d57740a03c8
```
Comment (larshesel) [inline changelog.md]: Hey @codeadict - seems something went wrong in the last commit :D

Comment (codeadict) [reply changelog.md]: Ohh wow sorry about that one, fixing

Comment (larshesel) [reply changelog.md]: no worries - can you put the new log line at the bottom of the list? makes it easier when cherry picking for bugfix-releases.

Comment (codeadict) [reply changelog.md]: Yup done @larshesel

**Comment by larshesel** [comment]
Wohoo! :)

**Pull request merged**